src/Functions/formatReadableTimeDelta.cpp

```diff
@@ -122,7 +122,7 @@ class FunctionFormatReadableTimeDelta : public IFunction
             throw Exception(ErrorCodes::BAD_ARGUMENTS,
                 "Unexpected value of maximum unit argument ({}) for function {}, the only allowed values are:"
                 " 'seconds', 'minutes', 'hours', 'days', 'months', 'years'.",
-                maximum_unit_str.toString(), getName());
+                std::string(maximum_unit_str), getName());
```

Comment:
It seems like cast to string is not necessary anymore :)

```suggestion
                maximum_unit_str, getName());
```